src/store/configureStore.dev.js

```diff
@@ -6,18 +6,16 @@ import { routerMiddleware } from 'connected-react-router';
 import { composeWithDevTools } from 'redux-devtools-extension/logOnlyInProduction';
 import { createLogger } from 'redux-logger';
 
-import apiClient from './apiClient';
-import createRootReducer from '../reducers/RootReducer';
-import rootSaga from '../sagas/RootSaga';
+import createRootReducer from '../reducers';
+import rootSaga from '../sagas';
 
-export default function configureStore() {
+export default function configureStore(initialState = {}) {
```

Comment:
This is a more canonical way of getting initial state into the store.  That way the store configuration doesn't know anything about apiClient/where data came from.